pom.xml

```diff
@@ -141,13 +145,18 @@
               <id>default-jar</id>
               <configuration>
                 <archive>
+                  <manifest>
+                    <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+                    <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
+                  </manifest>
                   <!-- Generate a jar INDEX.LIST -->
                   <index>true</index>
                   <!-- Add the Maven coordinates in the manifest -->
                   <manifestEntries>
                     <Maven-Group-Id>${project.groupId}</Maven-Group-Id>
                     <Maven-Artifact-Id>${project.artifactId}</Maven-Artifact-Id>
                     <Maven-Version>${project.version}</Maven-Version>
+                    <Implementation-URL></Implementation-URL>
```

Comment:
I think we can let the default value

Reply:
we should however customize this value per project

Reply:
so remove this line

Reply:
The default value when we remove the <Implementation-URL></Implementation-URL> is -
 Implementation-URL: http://nexus.sonatype.org/oss-repository-hosting.hml this comes from the org.sonatype.oss:oss-parent from vertx-parent

Agreed that each module needs to define its own 'url' to be complete.

I will make the changes